src/frontend/src/lib/utils/wizard-modal.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { WizardStepsSend } from '$lib/enums/wizard-steps';
+import { WizardModal, type WizardSteps } from '@dfinity/gix-components';
+
+export const goToWizardStep = ({
```

Comment:
Should we call it `goToWizardSendStep` maybe given that `stepName` is of type `WizardStepsSend`?